linchpin/provision/roles/azure/tasks/provision_azure_vm.yml

```diff
@@ -0,0 +1,77 @@
+---
+- name: "Set default res_grp_vars when not defined"
+  set_fact:
+    res_grp_vars:
+      - name: "{{ res_def['res_name'] | default(res_def['vm_name']) }}"
+        resource_group_name: "{{ res_grp_name  }}"
+
+# - name: Update Instance Tags
```

Comment:
@JunqiZhang0  do we need these comments ??

Reply:
No